charts/templates/deployment.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "link-shortener.fullname" . }}
+  namespace: link-shortener{{ .Values.django.env.run_env }}
+  labels:
+    {{- include "link-shortener.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "link-shortener.selectorLabels" . | nindent 6 }}
+  template:
```

Comment:
I recommend adding the following annotations such that the deployment will automatically trigger a redeploy if you update a configmap or secret.

```
  template:
    metadata:
      annotations:
        checksum/config: {{ include (print $.Template.BasePath "/link-shortener-configmap.yaml") . | sha256sum }}
        checksum/secrets: {{ include (print $.Template.BasePath "/link-shortener-secrets.yaml") . | sha256sum }}

Reply:
@cachemeoutside I'd prefer to wait on this change, and the restructuring you suggested with the Oral History examples, until after this application has been deployed.  For now, this application is consistent with the rest of our team's apps.  I'm happy to consider changes, but will need more time to understand them, and if we decide to adopt them, I'll want to update all of our applications to be consistent.